packages/harmony/src/components/button/Button.stories.tsx

```diff
@@ -4,7 +4,9 @@ import { Button } from './Button'
 import { ButtonProps, ButtonSize, ButtonType } from './types'
 
 const baseProps: ButtonProps = {
-  text: 'Click Me'
+  iconLeft: Icons.IconCampfire,
+  iconRight: Icons.IconCampfire,
```

Comment:
Make sure you merge main. I removed the icons and the `isLoading` examples below